yt_dlp/extractor/neteasemusic.py

```diff
@@ -66,45 +66,40 @@ def _download_eapi_json(self, path, video_id, query_body, headers={}, **kwargs):
                 **headers,
             }, **kwargs)
 
-    def _call_player_api(self, song_id, bitrate):
+    def _call_player_api(self, song_id, level):
         return self._download_eapi_json(
-            '/song/enhance/player/url', song_id, {'ids': f'[{song_id}]', 'br': bitrate},
-            note=f'Downloading song URL info: bitrate {bitrate}')
+            '/song/enhance/player/url/v1', song_id, {'ids': f'[{song_id}]', 'level': level, 'encodeType': 'flac'},
+            note=f'Downloading song URL info: level {level}')
 
-    def extract_formats(self, info):
-        err = 0
+    def _extract_formats(self, info):
         formats = []
+        format_MD5s = []
         song_id = info['id']
-        for song_format in self._FORMATS:
-            details = info.get(song_format)
-            if not details:
+        for level in self._LEVELS:
+            song = traverse_obj(self._call_player_api(song_id, level), ('data', 0, {dict})) or {}
+            if song.get('level') in [song_format.get('level') for song_format in formats]:
+                break  # API returns existing level - the user have no access to further ones
```

Comment:
this is wrong. there is no `level` field in the format dicts

Reply:
oof you're right. Fixed.